Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using static System.Math;
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            string cont = "y";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'cont'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4853259)

Reply:
This is a minor issue. Still line 12 will override the initial value and makes it unneccessary in the first place.